shared/wallets/routes.js

```diff
@@ -132,16 +132,6 @@ export default routeTree
 
 const sharedRoutes = {
   airdrop: {getScreen: () => require('./airdrop/container').default},
-  airdropQualify: {getScreen: () => require('./airdrop/qualify/container').default},
```

Comment:
looking at the stories, it looks like airdropQualify is a modal but airdrop isn't, but not 100% sure